tomography_preprocessing/denoising/care4relion/_utils.py

```diff
@@ -0,0 +1,107 @@
+from pathlib import Path
+from typing import Tuple, List, Dict
+
+import pandas as pd
+import starfile
+import typer
+import json
+import shutil
+
+def create_denoised_tomograms_dir(
+        output_directory: Path
+) -> Tuple[Path, Path, Path]:
+    training_dir = output_directory / 'external' / 'training' 
+    training_dir.mkdir(parents=True, exist_ok=True)
+    tomogram_dir = output_directory / 'tomograms'
+    tomogram_dir.mkdir(parents=True, exist_ok=True)
+    tilt_series_dir = output_directory / 'tilt_series'
+    tilt_series_dir.mkdir(parents=True, exist_ok=True)
+    return training_dir, tomogram_dir, tilt_series_dir
+    
+def parse_training_tomograms(
+        training_tomograms: str
+) -> List:
+    training_tomograms = training_tomograms.strip().split(':')
+    return training_tomograms
```

Comment:
in general I'm not a huge fan of this pattern for taking multiple CLI args - what's wrong with taking `List[str]` in the CLI?

Reply:
not sure how we would pass a python list from relion GUI onto cli and into script? str keeps it easy for now

Reply:
you could still take a string in RELION, you would just have to change how that string is turned into the CLI arg in `get_commands`/`getComands` where the whole call is generated... let's leave as is for now though